cmd/opfgd/start.go

```diff
@@ -81,8 +81,6 @@ func runStartCmd(ctx client.Context, cmd *cobra.Command, args []string) error {
 	// Create a cancellable context
 	fgCtx, cancel := context.WithCancel(context.Background())
 	defer cancel()
-	// Start monitoring BTC staking activation
-	go fg.MonitorBtcStakingActivation(fgCtx)
```

Comment:
can we add a TODO here and refer to the Github issue https://github.com/babylonlabs-io/rollup-finality-gadget/issues/52?

Reply:
Do we need this method at all?